### PR TITLE
bind: update to 9.14.4

### PR DESCRIPTION
**Pull request by dengqf6**
Maintainer: @nmeyerhans 
Compile tested: ramips, ath79
Run tested: ramips, ath79

Description:
Update bind to 9.14.4

### CODE REVIEW

**Comment by hnyman** [comment]
@nmeyerhans 
Any objections?

File: net/bind/Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:= \
 	https://www.mirrorservice.org/sites/ftp.isc.org/isc/bind9/$(PKG_VERSION) \
```
Comment (diizzyy) [inline net/bind/Makefile]: Replace with officially listed mirrors (https://ftp.isc.org/isc/MIRRORS)
https://ftp.funet.fi/pub/mirrors/ftp.isc.org/isc/bind9/
http://ftp.iij.ad.jp/pub/network/isc/bind9/
http://isc.mirrors.pair.com/bind9/

Comment (nmeyerhans) [reply net/bind/Makefile]: That's a good idea, but we've been using the current mirror list for long enough that I wouldn't reject this PR for it.

**Comment by diizzyy** [comment]
https://github.com/openwrt/packages/blob/master/net/bind/Makefile#L37
Can't that be replaced with: ```DEPENDS: +BIND_LIBXML2:libxml2``` and so on?

https://github.com/openwrt/packages/blob/master/net/bind/Makefile#L140
Replace with ```$(if $(CONFIG_BIND_LIBJSON),--with-libjson="$(STAGING_DIR)/usr",--without-libjson)``` and so on?
I guess you should be able to drop the paths too but I haven't tested

**Pull request merged**